### PR TITLE
(#3) Added new Swarm interface with inspect() method

### PR DESCRIPTION
**Pull request by llorllale**
This PR:
* solves #3 
* Uses the [Docker Swarm API Docs](https://docs.docker.com/engine/api/v1.35/#tag/Swarm) as reference
* Adds new `Swarm` with `inspect()` method
* Leaves puzzle to finish adding and impementing the rest of the Swarm methods
* Leaves puzzle to finish implementing the rest of the Docker methods (except for Swarm)
* Leaves puzzle for a refactoring of "inspect" operations in general

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#66](https://github.com/amihaiemil/docker-java-api/issues/66) is now in scope, role is `REV`

File: src/main/java/com/amihaiemil/docker/Docker.java [hunk 1]
```diff
@@ -74,4 +72,9 @@
      */
     Exec exec();
 
+    /**
+     * Entry point fr the Swarm API.
```
Comment (amihaiemil) [inline src/main/java/com/amihaiemil/docker/Docker.java]: @llorllale typo here

File: src/main/java/com/amihaiemil/docker/Docker.java [hunk 2]
```diff
@@ -32,8 +32,6 @@
  * @author Mihai Andronache (amihaiemil@gmail.com)
  * @version $Id$
  * @since 0.0.1
- * @todo #1:30min Start implementing the other parts of the API,
```
Comment (amihaiemil) [inline src/main/java/com/amihaiemil/docker/Docker.java]: @llorllale I think you have to leave another puzzle for continuing implementation with, say, ``Images`` API.

File: src/main/java/com/amihaiemil/docker/Docker.java (hunk 1 above)
Comment (llorllale) [reply src/main/java/com/amihaiemil/docker/Docker.java]: @amihaiemil ok

File: src/main/java/com/amihaiemil/docker/Docker.java (hunk 2 above)
Comment (llorllale) [reply src/main/java/com/amihaiemil/docker/Docker.java]: @amihaiemil yes

**Comment by llorllale** [comment]
@amihaiemil done, please review

**Comment by amihaiemil** [comment]
@rultor merge it

**Comment by amihaiemil** [comment]
@rultor merge again...

**Comment by amihaiemil** [comment]
@llorllale let's try again later... Rultor has a bug, it tries to sign/decrypt the release config files even though the command is not ``release``. And, apparently, the gpg server is down, as we see in the logs.

**Comment by amihaiemil** [comment]
@llorllale I actually forgot I can merge manually haha

**Comment by 0crat** [comment]
The job [#66](https://github.com/amihaiemil/docker-java-api/issues/66) is now out of scope

**Pull request merged**